srcpkgs/libraw/template

```diff
@@ -1,12 +1,12 @@
 # Template file for 'libraw'
 pkgname=libraw
 version=0.21.1
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="pkg-config automake libtool"
-makedepends="jasper-devel lcms2-devel"
+makedepends="jasper-devel lcms2-devel libgomp-devel"
 short_desc="Raw image decoder library"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Daniel Martinez <danielmartinez@airmail.cc>"
```

Comment:
Please delay the adoption. [We prefer to establish a history of maintenance](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#adopting-a-template) before allowing people to claim packages.